tests/bootstrap.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /*
- * Copyright 2014 Google Inc.
+ * Copyright 2015 Google Inc.
```

Comment:
It should stay at the year when it was first written.
http://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated